.drone.yml

```diff
@@ -386,7 +386,23 @@ services:
         path: /tmp
 
 steps:
-  - name: gce
+  - name: osctl-cluster-create
```

Comment:
Broke the osctl cluster into its own step. The GCE and Azure e2e tests will both use the same CAPI deployment on this cluster for their testing.